examples/express/app.js

```diff
@@ -2,8 +2,10 @@ const Bugsnag = require('@bugsnag/js')
 const express = require('express')
 const { readFileSync } = require('fs')
 
-Bugsnag.start(process.env.BUGSNAG_API_KEY)
-Bugsnag.use(require('@bugsnag/plugin-express'))
+Bugsnag.start({
+  apiKey: process.env.BUGSNAG_API_KEY,
+  plugins: [require('@bugsnag/plugin-express')]
+})
```

Comment:
Is there anything wrong with doing this?

```suggestion
const BugsnagExpressPlugin = require('@bugsnag/plugin-express')

Bugsnag.start({
  apiKey: process.env.BUGSNAG_API_KEY,
  plugins: [BugsnagExpressPlugin]
})
```

This maps more to the structure of the docs (with the requires/imports going first) and, to my mind, reads better.

Reply:
Done